msig_contract_deployment/msig_erc20_account_creation.md

```diff
@@ -0,0 +1,33 @@
+<h1>Multi-sig account creation of evm accounts for EOS-ERC20 token bridge </h1>
+
+<h2>Overview</h2>
+EOS-EVM will soon support trustless token bridge between tokens on native EOS (for example: USDT on EOS) and ERC-20 compatible tokens on EOS-EVM. (contract implementaion can be found in https://github.com/eosnetworkfoundation/evm-bridge-contracts). Because of such, we need the multi-sig from 21 block producers to create the following accounts on testnet & mainnet:
+
+<h3>account <b>eosio.erc2o</b></h3>
```

Comment:
By the way, you don't have to use HTML for the headings. In Markdown you can use `#`, `##`, etc. to do headings and sub-headings.

For example, the title could be:
```
# Multi-sig account creation of accounts needed for EOS EVM ERC20 token bridge
```

The Overview sub-heading, which doesn't even really seem to be necessary since there is no other `h2` heading in this document could be:

```
## Overview
```

And then this `h3` heading could be:

```
### Account eosio.erc2o
```